go/exocom-mock/bin/spec.cmd

```diff
@@ -1,2 +0,0 @@
-go install
```

Comment:
Until we have a regular windows developer, I don't know if its worth keeping those up to date. If we added appveyor ci, that would be a good way to force us to keep those up to date